i18n/locales/en/translations.ts

```diff
@@ -5,15 +5,15 @@ export const translations = {
     chapter: 'Chapter',
     challenge: 'Challenge',
     coming_soon: 'Coming soon',
-    start_chapter: 'Start chapter',
+    start_chapter: 'Start chapter'
   },
 
   chapter_one: {
     title: 'Secrets in plain sight',
     paragraph_one:
-      'The year is 2139. The very last bitcoin is slated to be mined in two weeks. For months, a final countdown has been taking place in Times Square for the last block when there will be officially no more bitcoin issued.',
-    paragraph_two: `And then suddenly, the network comes to a crawl. Instead of the normal ten-minute block interval, blocks are coming in just a few times a day, if not longer. Your mom's bitcoin node stops syncing. The media starts reporting on the situation. It's everywhere on social media.`,
-    paragraph_three: `The price of bitcoin is falling. There is a knock at your door, and when you open it, a yellowed, age-weathered envelope has been left for you. There's no name or return address. What could be inside?`,
+      'The year is 2139. The last bitcoin is two weeks from being mined. For months, a clock has ticked down in Satoshi Square. The world awaits the last block. Then, suddenly, the network grinds to a halt.',
+      paragraph_two: ' ',
+      paragraph_three: ' '
```

Comment:
Instead of these translations being blank, we should be able to remove the content in `/content/chapters/chapter-1/index.ts` to match this format
```tsx
<Chapter metadata={metadata} lang={lang}>
      <Text>{t('chapter_one.paragraph_one')}</Text>
</Chapter>
```
This will not have any implications for limiting future chapters as this is inside the chapter-1 content

Reply:
Fixed, thanks. I could actually not figure out where the template for this was defined until you pointed it out here.